WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -188,15 +188,28 @@ $venue-map-wrapper-height: 400px;
   }
 }
 
-#simple-nav {
+$comp-list-nav-height: 39px;
+#comp-list-nav {
   z-index: 10;
   background-color: #eee;
-  line-height: 39px;
+  line-height: $comp-list-nav-height;
 
   &.affix {
     top: 0;
     border-bottom: 1px solid $gray-base;
   }
+
+  .affix + .comp-list-nav-placeholder {
```

Comment:
Are you *sure* this works? Are you sure you don't want `&.affix + .comp-list-nav-placeholder` here instead?